README.md

```diff
@@ -94,6 +94,19 @@ Once the gamer is connected to Passport, the SDK will store your credentials (ac
 
 See this Blueprint showing how to logout from passport ![Passport Logout Blueprint](PassportLogoutFlow.jpg)
 
+#### Imx Transfer
+
+> Note: The transfers feature require pre-approval from Immutable. Please reach out to us before making use of it.
```

Comment:
Can we add a link to contact us, too? To https://docs.immutable.com/docs/x/contact/

Like how we have it on the Unity SDK official docs https://docs.immutable.com/docs/x/sdks/unity#immutable-x-transfer